src/plugins/opensearch_dashboards_overview/public/components/overview/overview.tsx

```diff
@@ -149,10 +149,12 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features, bran
         hideToolbar={isNewOpenSearchDashboardsInstance}
         iconType="inputOutput"
         title={
-          <FormattedMessage
-            defaultMessage="OpenSearch Dashboards"
-            id="opensearchDashboardsOverview.header.title"
-          />
+          branding.applicationTitle || (
+            <FormattedMessage
+              defaultMessage="OpenSearch Dashboards"
+              id="opensearchDashboardsOverview.header.title"
+            />
+          )
```

Comment:
What do you think of the above suggestion? It is aligned with previous application title configuration: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/6b42669f1168644806a9d4231077746405623c6a/src/plugins/home/public/application/components/welcome.tsx#L237

Reply:
Its sounds great